resources/certificates/files/gardener.sh

```diff
@@ -3,6 +3,8 @@ set -eo pipefail
 
 export HOME="/tmp"
 
+CERT_CHECK_TIMEOUT=5
```

Comment:
It could be a configurable env at the `job.yaml` level definition.

Reply:
ptal at newest version.